pythreepio/pythreepio/utils.py

```diff
@@ -1,11 +1,15 @@
+#importlib.resources is imported  to provide access to resources within packages
 try:
     import importlib.resources as pkg_resources
+#for easier accessibility  importlib_resources can be refered to as pkg_resources
 except ImportError:
     import importlib_resources as pkg_resources
 import json
 import static
+#static is imported to Serve static or templated content via WSGI 
 
-
+#this function is used to read data from "mapped_commands_full.json" file and returns an  dictionary  based on JSON file
```

Comment:
Could you document this function according to the [spinx documentation standard](https://pythonhosted.org/an_example_pypi_project/sphinx.html#function-definitions)? This will allow us to autogenerate documentation based on the comments :)